src/index.html

```diff
@@ -13,5 +13,182 @@
   <body>
     <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      name="html form">
+
+      <fieldset >
```

Comment:
Don't add spaces between the parent and the child component
```suggestion
      <fieldset >
```